core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -1221,6 +1222,20 @@ public void testReplicaManagerFetchShouldProceed() {
             any(), any(), any(ReplicaQuota.class), any());
     }
 
+    @Test
+    public void testCloseSharePartitionManager() throws Exception {
+        Timer timer = Mockito.mock(SystemTimerReaper.class);
+        SharePartitionManager sharePartitionManager = SharePartitionManagerBuilder.builder().
+                withTimer(timer).build();
```

Comment:
You might also want to pass the mock persister and verify if close has been called on that as well.